torch/_ops.py

```diff
@@ -223,6 +223,8 @@ def __getattr__(self, op_name):
         # It is not a valid op_name when __file__ is passed in
         if op_name == "__file__":
             return "torch.ops"
+        elif op_name == "__origin__":
```

Comment:
__origin__ refers to what.. the base type that `_OpNamespace` extends from, e.g. (types.ModuleType)?

Why does this change help? does torch._C._get_jit_operation currently raise AttributeError for the __origin__ case or does that actually run through for some reason currently?

Reply:
On Generic types, `__origin__` points to the non-parameterized generic class. Other types don't have this attribute. JIT code attempts to query if the `__origin__` attribute exists, falling back if it does not. This is ordinarily harmless, but for `torch._OpNamespace` this triggers the `__getattr__` here. This getattr will then go into C++ and attempt to look up the operator, raising a C++ exception when it fails to do so. This ends up having the same net result, but it is annoying for gdb users as it triggers `catch throw`. By intercepting it earlier, we can avoid the raise.